.github/CODEOWNERS

```diff
@@ -86,6 +86,7 @@
 /avm/res/desktop-virtualization/scaling-plan/ @Azure/avm-res-desktopvirtualization-scalingplan-module-owners-bicep @Azure/avm-module-reviewers-bicep
 /avm/res/desktop-virtualization/workspace/ @Azure/avm-res-desktopvirtualization-workspace-module-owners-bicep @Azure/avm-module-reviewers-bicep
 /avm/res/dev-ops-infrastructure/pool/ @Azure/avm-res-devopsinfrastructure-pool-module-owners-bicep @Azure/avm-module-reviewers-bicep
+/avm/res/dev-center/network-connection @Azure/avm-res-devcenter-networkconnection-module-owners-bicep @Azure/avm-module-reviewers-bicep
```

Comment:
```suggestion
/avm/res/dev-center/network-connection/ @Azure/avm-res-devcenter-networkconnection-module-owners-bicep @Azure/avm-module-reviewers-bicep
```

Reply:
Not sure why I can't see the change here, because I can see it in your last workflow run: https://github.com/ahmadabdalla/bicep-registry-modules/commit/4743f79633f38f605d9531fdcf8a3e57dee575d9

Reply:
yeah its already in

Reply:
can see it now. The team is not confirmed in the hierachy yet though. Already requested it to be added to the AVM owners team? If the case we just need to wait for @matebarabas to approve.

Once both happened, this error/warning will go away 
![image](https://github.com/user-attachments/assets/29af8a73-9551-4533-9c10-6843bf9b2dfc)

Reply:
Damn, it was very unfortunate that this comment was resolved @ahmadabdalla 😄 The teams are **not** ready and the PR was on auto-complete. This will cause some issues down the line for future PRs as now the upstream codeowners file has the error to. This is my bad as I should have unresolved this comment.
Please make sure to get the teams created/added to the hierachy asap. Thank you 🙏

Reply:
Done! 
https://github.com/orgs/Azure/teams/avm-res-devcenter-networkconnection-module-owners-bicep

https://github.com/orgs/Azure/teams/avm-res-devcenter-networkconnection-module-contributors-bicep